<!-- -*- mode: XML -*- -->

<launch>
    <!-- move_arm -->
    <node name="move_left_arm" pkg="move_arm" type="move_arm_simple_action" output="screen" machine="c2">
        <param name="group" type="string" value="left_arm" />
        <remap from="arm_ik" to="wubble2_left_arm_kinematics/get_constraint_aware_ik" />
        <param name="controller_action_name" type="string" value="/l_arm_controller/follow_joint_trajectory" />
    </node>
</launch>
